src/components/MultiSelectGroup/MultiSelectGroup.stories.tsx

```diff
@@ -0,0 +1,295 @@
+        multiSelectItems.map((group) => (
+          <MultiSelect
+            key={group.id}
+            id={group.id}
+            label={group.name}
+            variant="dialog"
+            items={group.items}
+            selectedItems={selectedItems}
+            onChange={(e) => {
+              handleChange(e.target.id, group.id);
+            }}
+            onMixedStateChange={(e) => {
+              const childItems = group.items
+                .filter((items) => items.id === e.target.id)[0]
+                .children.map((child) => child.id);
+              return handleMixedStateChange(group.id, childItems);
+            }}
+            onClear={() => {
+              handleClear(group.id);
+            }}
+            onApply={() => {}}
+          />
+        ))}
+    </MultiSelectGroup>
+  );
+};
+
+export const MultiSelectGroupLayoutStory: Story<MultiSelectGroupProps> = () => {
+  const [selectedItems, setSelectedItems] = React.useState({});
+
+  function handleChange(itemId: string, groupId: string) {
+    let itemIds;
+    // Check if the id already exists in the state
+    if (
+      selectedItems[groupId] !== undefined &&
+      // @TODO Temporary hack to make availability multiselect use radios.
+      groupId !== "availability"
+    ) {
+      let itemIdExists = selectedItems[groupId].items.indexOf(itemId) > -1;
+      // Make a copy of the existing array.
+      itemIds = selectedItems[groupId].items.slice();
+      // If termId exists, remove it from the array.
+      if (itemIdExists) {
+        itemIds = itemIds.filter((id) => id !== itemId);
+      } else {
+        // Add it to the array, but modify the copy, not the original.
+        itemIds.push(itemId);
+      }
+    } else {
+      itemIds = [];
+      itemIds.push(itemId);
+    }
+    setSelectedItems({
+      ...selectedItems,
+      [groupId]: {
+        items: itemIds,
+      },
+    });
+  }
+
+  function handleMixedStateChange(groupId: string, childItems: string[]) {
+    let newItems;
+    // Some selected items for group already exist in state.
+    if (selectedItems[groupId] !== undefined) {
+      //
+      if (
+        childItems.every((childItem) =>
+          selectedItems[groupId].items.includes(childItem)
+        )
+      ) {
+        newItems = selectedItems[groupId].items.filter(
+          (stateItem) => !childItems.includes(stateItem)
+        );
+      } else {
+        // Merge all child items.
+        newItems = [...childItems, ...selectedItems[groupId].items];
+      }
+    } else {
+      newItems = childItems;
+    }
+
+    setSelectedItems({
+      ...selectedItems,
+      [groupId]: {
+        items: newItems,
+      },
+    });
+  }
+
+  function handleClear(groupId: string) {
+    let newSelectedItems = {};
+    for (let key of Object.keys(selectedItems)) {
+      if (key !== groupId) {
+        newSelectedItems[key] = selectedItems[key];
+      }
+    }
+    setSelectedItems(newSelectedItems);
+  }
+  return (
+    <>
```

Comment:
Instead of a fragment and separating with a `<br />`, use Chakra's `VStack`:
```
<VStack gap="grid.m" align="start">
    <MultiSelectGroup ... ></MultiSelectGroup>
    <MultiSelectGroup ... ></MultiSelectGroup>
</VStack>
```

Reply:
updated